test/scenarios/17. settings/advanced/show-test-networks.csv

```diff
@@ -0,0 +1,12 @@
+Advanced settings: Check “Show test networks” toggle works correctly,,,,
+Step,Test Steps,Test Data,Expected Result,Notes
+1,Open the extension.,,The Welcome Back screen is shown,
+2,Proceed to Unlock the wallet.,password (8 characters min),,
+3,Click on the right corner setting(three dots) and click on the Settings from menu.,,Settings page appears,
```

Comment:
We may want to split this into two steps.  step three to click on 3 dots menu, step 4 could be to select settings.

Reply:
done